ble_controller/doc/api.rst

```diff
@@ -3,52 +3,41 @@
 API documentation
 #################
 
-
 BLE Controller
 **************
 
-.. doxygengroup:: blectlr
+.. doxygengroup:: ble_controller
    :project: nrfxlib
    :members:
 
 
 BLE Controller HCI
 ******************
 
-.. doxygengroup:: blectlr_hci
+.. doxygengroup:: ble_controller_hci
    :project: nrfxlib
    :members:
 
 
-BLE Controller utilities
+BLE Controller SoC
 ************************
 
-.. doxygengroup:: blectlr_util
+.. doxygengroup:: ble_controller_soc
    :project: nrfxlib
    :members:
 
-nRF Errors
-**********
-
-.. doxygengroup:: NRF_ERRORS_BASE
-   :project: nrfxlib
-   :members:
 
+Timeslot
+********
 
-.. doxygengroup:: NRF_ERRORS
+.. doxygengroup:: ble_controller_timeslot
    :project: nrfxlib
    :members:
 
-nRF SoC
-*******
 
-.. doxygengroup:: NRF_SOC_DEFINES
-   :project: nrfxlib
-   :members:
+nRF errno
```

Comment:
I have removed the errno documentation from here as a workaround for the duplication warnings in the documentation build. The header is more or less a duplication of the one in bsdlib, but since we want the different artifacts to be self-contained release bundles, it is a copy.